README.md

```diff
@@ -259,7 +259,7 @@ A language is missing? Want to improve this cheat sheet? [Contributions are welc
 
 ## ❤️ Contributors
 
-Thanks goes to these wonderful people:
+Thanks go to these wonderful people:
```

Comment:
```suggestion
Thanks goes to these wonderful people:
```

Reply:
Je me suis basé sur ça : https://english.stackexchange.com/questions/15429/which-is-correct-special-thanks-go-to-or-special-thanks-goes-to